bin/graft.jl

```diff
@@ -20,42 +22,70 @@ vdsre = findassign(micro1def, :dS)[1]
 vdsre = replacevar(vdsre, :K,:N)
 @show popgrowth = vdsre.args[2].args[2]
 
+
+model_1 = ModelTool.model(ModelTool.ExpODEProblem, model1)
+model_2 = ModelTool.model(ModelTool.ExpODEProblem, model2)
+lhs(x::Expr) = begin
+    @show x
+    x.head == :(=) || error("x is not an assignment")
+    return x.args[1]
+end
+fluxes(x::ModelTool.ExpODEProblem) = x.variables[1].flux
+
+# fluxvar = lhs(fluxes(model_2)[1].args[2])
+fluxvar = fluxes(model_2)[1].args[2].args[1]
+popgrowth = replacevar(findassign(model_2.funcs[1], fluxvar)[1], :K, :N).args[2].args[2]
+ex = model_1.variables[1].flux[1]
+# TODO: add N=sum(S,E,I)
+ex.args[2] = :($(popgrowth)+$(ex.args[2]))
```

Comment:
On the same note, the single `.args[2]` calls I think are alright because they are just the expression for a known call. You can come up with that pretty easily  without dumping the expression and counting.

Reply:
Well I'm thinking the API would be more like `addterm!(ex, +, popgrowth, :S)` which would do the finding and replacing. So that our API would be more semantic rather than lexical.